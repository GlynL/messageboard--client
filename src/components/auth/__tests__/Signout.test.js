import React from "react";
import { shallow } from "enzyme";
import { Signout } from "../Signout";

let wrapper, signout;
beforeEach(() => {
  signout = jest.fn();
  wrapper = shallow(<Signout signout={signout} />);
});

it("renders Signout correctly", () => {
  expect(wrapper).toMatchSnapshot();
});

it("handles signout call in componentDidMount", () => {
  expect(signout).toHaveBeenCalled();
});
