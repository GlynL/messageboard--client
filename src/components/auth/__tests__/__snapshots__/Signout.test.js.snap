// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Signout correctly 1`] = `
<div>
  Sorry to see you go.
</div>
`;
